pkg/generators/markers_test.go

```diff
@@ -131,20 +131,102 @@ func TestParseCommentTags(t *testing.T) {
 		},
 		{
 
-			t:    &structKind,
+			t:    structKind,
 			name: "invalid: invalid value",
 			comments: []string{
 				"+k8s:validation:",
 			},
 			expectedError: `failed to parse marker comments: cannot have empty key for marker comment`,
 		},
 		{
-			t:    &numKind,
+			t: types.Float64,
+			// temporary test. ref support may be added in the future
```

Comment:
noop? Might be good to add something else to the test case to ensure we have an output and the ref is ignored.

Reply:
not a no-op, just the expected output is the empty `CommentTags`, so we are checking that the ref is ignored